src/commands/sfpowerkit/org/sandbox/refresh.ts

```diff
@@ -31,6 +34,11 @@ export default class Refresh extends SFPowerkitCommand {
       char: "f",
       default: "",
       description: messages.getMessage("cloneFromFlagDescripton")
+    }), 
+    description: flags.string({
+      required: false,
+      char: "d",
+      description: messages.getMessage("descriptionFlagDescription")
```

Comment:
@Janani-26  flag description for descriptionFlagDescription should be added in [/messages/sandbox_refresh.json/messages/sandbox_refresh.json](https://github.com/Accenture/sfpowerkit/blob/main/messages/sandbox_refresh.json). 


Rest all looking good we will update the read me with new flags and help

Reply:
my bad flag description already exist. 

all good from my side.